app/templates/index.hbs

```diff
@@ -71,12 +71,14 @@
       {{/layout.blowout}}
 
       {{#layout.cards as |row|}}
-        {{row.brick item=model.wqxrHome.featuredItems.[3]}}
-        {{row.brick item=model.wqxrHome.featuredItems.[4]}}
-        {{row.brick item=model.wqxrHome.featuredItems.[5]}}
+        {{#each (slice model.wqxrHome.featuredItems 3 6) as |item|}}
+          {{row.brick item=item}}
+        {{/each}}
       {{/layout.cards}}
 
-      {{layout.blowout flip=true items=(slice model.wqxrHome.featuredItems 6 9)}}
+      {{#if (gte model.wqxrHome.featuredItems.length 9)}}
+        {{layout.blowout flip=true items=(slice model.wqxrHome.featuredItems 6 9)}}
+      {{/if}}
```

Comment:
I don't get why we would render items six through eight if the check for nine or more items is true?

Reply:
This is a blowout, so i thought it would be best to simply not show the whole thing if only some of the items were there. The indexing starts at zero, so this it will actually render items 7 through 9.

Reply:
ah, ok. that makes sense to me.